hazelcast/src/main/java/com/hazelcast/nio/serialization/GenericRecordBuilder.java

```diff
@@ -68,775 +80,1025 @@ static GenericRecordBuilder compact(String typeName) {
-     * Otherwise, it is illegal to set to the same field twice.
+     * It is legal to set the field again only when Builder is created with
+     * {@link GenericRecord#newBuilderWithClone()}. Otherwise, it is illegal to
+     * set to the same field twice.
      *
-     * @param fieldName name of the field as it is defined in its class definition.
-     *                  See {@link ClassDefinition} for {@link Portable}
+     * @param fieldName name of the field as it is defined in its schema/class
+     *                  definition. It should be composed of only alphanumeric
+     *                  characters for {@link Portable} GenericRecords.
      * @param value     to set to GenericRecord
      * @return itself for chaining
-     * @throws HazelcastSerializationException if the field name does not exist in the class definition or
-     *                                         the type of the field does not match the one in the class definition or
-     *                                         Same field is trying to be set without using
-     *                                         {@link GenericRecord#cloneWithBuilder()}.
+     * @throws HazelcastSerializationException if the field name does not exist
+     *                                         in the schema/class definition or
+     *                                         the type of the field does not
+     *                                         match the one in the schema/class
+     *                                         definition or the same field is
+     *                                         trying to be set without using
+     *                                         {@link
+     *                                         GenericRecord#newBuilderWithClone()}.
      */
     @Nonnull
     GenericRecordBuilder setFloat64(@Nonnull String fieldName, double value);
 
     /**
-     * It is legal to set the field again only when Builder is created with {@link GenericRecord#cloneWithBuilder()}.
-     * Otherwise, it is illegal to set to the same field twice.
+     * It is legal to set the field again only when Builder is created with
+     * {@link GenericRecord#newBuilderWithClone()}. Otherwise, it is illegal to
+     * set to the same field twice.
      * <p>
-     * Supported only for {@link com.hazelcast.config.CompactSerializationConfig Compact}. Not applicable to {@link Portable}.
+     * Supported only for
+     * {@link com.hazelcast.config.CompactSerializationConfig Compact}. Not
+     * applicable to {@link Portable}.
      *
-     * @param fieldName name of the field as it is defined in its class definition.
-     *                  It should be composed of only alpha-numeric characters.
-     *                  See {@link ClassDefinition} for {@link Portable}
+     * @param fieldName name of the field as it is defined in its class
+     *                  definition.
```

Comment:
```suggestion
     * @param fieldName name of the field as it is defined in its schema.
     *                  
```

Similar for other setNullableXX methods